packages/component-library/src/components/plmg-page-container/plmg-page-container.scss

```diff
@@ -9,7 +9,7 @@
   .plmg-content {
     display: block;
     position: absolute;
-    height: 100%;
+    height: calc(100% - 99px); //Height of the header
```

Comment:
@jamesducky @JakeH91 @nathan-bird
About using variables in our css...

The [page header component](https://github.com/DuckyTeam/Plumage/blob/main/packages/component-library/src/components/plmg-header/plmg-header.scss#L18) uses the height of the top of the sidebar to determine its own height, and I'm using the height of the header to determine the offset for the height of the page content container.
These are all hardcoded in every scss file, so if we change the height of one of those components, the heights elsewhere will not be changed. We could use variables?
I think I'd either have to add a variable to `global.scss` for that height value, and then it'll be available to all scss files, which seems odd. Or, we could split up global.scss to be
```
/styles
|--/global
|  |-- plumage-tokens.scss
|--/components
|  |-- component-name.scss
```
Or something like that. Which might be overkill and stencil has this [globalStyle](https://github.com/DuckyTeam/Plumage/blob/main/packages/component-library/stencil.config.ts#L7) property in the config which only takes one file... Which makes me lean towards just using the file we already have. Any preference here?

Reply:
I'd agree that simpler is better. Shall we move that conversation to a separate task in Asana?